subjects/lastword/README.md

```diff
@@ -2,24 +2,48 @@
 
 ### Instructions
 
-Write a program that takes a `string` and displays its last word, followed by a newline (`'\n'`).
+Write a function `LastWord that takes a `string`and returns its last word followed by a`\n`.
 
 - A word is a section of `string` delimited by spaces or by the start/end of the `string`.
 
-- The output will be followed by a newline (`'\n'`).
+### Expected function
 
-- If the number of arguments is different from 1, or if there are no word, the program displays nothing.
+```go
+func LastWord(s string) string{
+
+}
+```
 
 ### Usage
 
+Here is a possible program to test your function :
+
+```go
+package main
+
+import (
+	"piscine"
+
+	"github.com/01-edu/z01"
+)
+
+func main() {
+	z01.PrintRune(piscine.LastWord("this        ...       is sparta, then again, maybe    not"))
```

Comment:
here you can use "fmt" if you want it to be more clean, otherwise just close this conv its fine like this as well

Reply:
I'll change it with `fmt.Print`